pkg/scheduler/scheduler.go

```diff
@@ -46,6 +47,10 @@ import (
 	internalqueue "k8s.io/kubernetes/pkg/scheduler/internal/queue"
 	"k8s.io/kubernetes/pkg/scheduler/metrics"
 	"k8s.io/kubernetes/pkg/scheduler/profile"
+
+	"go.opentelemetry.io/otel/exporters/trace/jaeger"
```

Comment:
just a note for the future: once otel collector exporter will be released, it will be better to switch to it.

Reply:
+1 that was my ultimate goal, but for this demo I just wanted to have something working (I had some trouble exporting with the otel collector locally)